Makefile

```diff
@@ -0,0 +1,34 @@
+# Get protoc commands
+PROTOC := $(shell command -v protoc)
+PROTOC_GEN_GO := $(shell command -v protoc-gen-go)
+PROTOC_GEN_GO_GRPC := $(shell command -v protoc-gen-go-grpc)
+
+# Get package manager
+PACKAGE_MGR := $(shell ([ -f /etc/redhat-release ] && echo yum) || ([ -f /etc/debian_version ] && echo apt-get))
+
+all: checkenv build
+
+build: checkenv
+	protoc --go_out=lib/go/ *.proto
```

Comment:
lib/go/replication replication.proto ? because of https://github.com/kube-storage/spec/pull/9

Reply:
Done